lib/klaviyo/client.rb

```diff
@@ -63,9 +63,9 @@ def self.build_params(params)
       "data=#{Base64.encode64(JSON.generate(params)).gsub(/\n/,'')}"
     end
 
-    def self.check_email_or_id_exists(kwargs)
-      if kwargs[:email].to_s.empty? and kwargs[:id].to_s.empty?
-        raise Klaviyo::KlaviyoError.new(NO_ID_OR_EMAIL_ERROR)
+    def self.check_required_args(kwargs)
```

Comment:
Thoughts on making this method name more descriptive? eg check_required_identifier_args